charts/posthog/templates/web-deployment.yaml

```diff
@@ -69,6 +69,8 @@ spec:
         {{- include "snippet.redis-env" . | indent 8 }}
         - name: SITE_URL
           value: {{ template "posthog.site.url" . }}
+        - name: STATIC_URL
```

Comment:
I'd also add this to events-deployment if only to keep the env vars in sync to avoid copy-pasta errors down the line :)

Reply:
Yup will check where else this might be relevant, I guess just anywhere we're running the django app